client/homescreen/layout.js

```diff
@@ -71,6 +75,8 @@ export const Layout = ( {
 
 	const isTaskListEnabled = bothTaskListsHidden === false;
 	const isDashboardShown = ! query.task;
+	const isRunningTwoColumnExperiment =
+		process.env.JEST_WORKER_ID === undefined;
```

Comment:
This is a temporary change until we implement ExPlat.

I used `process.env.JEST_WORKER_ID === undefined` so that the `client/homescreen/test/index.js` test uses the original tasklist when running the Jest test.